### PR TITLE
Feature/status command

### LINKED ISSUE #19: Status command
Add some kind of status (might need better naming) command in CLI when user will be displayed:

 - node URL
 - actor address
 - owner's address (same one used to track balance)
 - datetime of last sent report

### PR DESCRIPTION
**Pull request by MakMuftic**
Closes #19 

I also refactored commands code in this PR so each command is in a separate file.

### CODE REVIEW

File: cmd/hactar/commands/status.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+)
+
+type StatusParams struct {
+	cli.Helper
+}
+
+func RunStatusCommand(ctx *cli.Context) error {
+	lotusClient, err := lotus.NewClient(nil, nil)
+	if err != nil {
+		fmt.Println("Unable to initialize lotus client")    // show to user
```
Comment (morrigan) [inline cmd/hactar/commands/status.go]: detect* This sound like we are starting one and might be confusing to our dummy users

File: cmd/hactar/commands/status.go [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+		log.Error("Failed to initialize lotus client", err) // log
+		return err
+	}
+	actorAddress, err := lotusClient.Miner.GetMinerAddress()
+	if err != nil {
+		fmt.Println("Lotus worker down")    // show to user
```
Comment (morrigan) [inline cmd/hactar/commands/status.go]: Also be more precise "miner worker*"

File: cmd/hactar/commands/status.go [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+		fmt.Println("Unable to get owner's address")       // show to user
+		log.Error("Failed to get owner's default address") // log
+		return err
+	}
+
+	fmt.Printf("Node url: %s\n", url.GetUrl())
+	fmt.Printf("Actor address: %s\n", actorAddress)
+	fmt.Printf("Owner address: %s\n", defaultAddress)
+	fmt.Printf(
+		"Last reported tipset of height %d at %s",
```
Comment (morrigan) [inline cmd/hactar/commands/status.go]: You can put something like "Monitoring block rewards, last reported tipset is..."

File: cmd/hactar/commands/status.go [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+		session.CurrentSession.GetLastCheckedHeight(),
+		session.CurrentSession.GetLastCheckedHeightTimestamp())
+
```
Comment (morrigan) [inline cmd/hactar/commands/status.go:61]: Add another message "Reporting other stats on the following intervals:" and display the intervals

**Review by morrigan — CHANGES_REQUESTED** [review]
Looks great! Thanks for adding the timestamp info, it inspired me for something more :D

**Comment by MakMuftic** [comment]
@morrigan this is sample output of status command

```
Hactar daemon app stats
-----------------------
Actor address:  t0101
Node url:       192.168.0.32
Owner address:  t3ue7o3jzopnzh2rfivxywbqn7xc3i34ng7sepmn2btfh4scod4a2ebu3oiyhhwnlpqd2noxdasozwyh2wbjea

Monitoring block rewards
------------------------
Last reported tipset height:  0
Last tipset reported at:      2020-02-03 12:20:32      

Reporting stats on the following intervals (seconds):
-----------------------------------------------------
General info report  60
Uptime report        10
Block reward report  50
Balance report       150

```

**Review by morrigan — APPROVED** [review]
Looks good!

FYI: UX would be better if you would read naturally "60 seconds" and not just the numbers listed :)

**Pull request merged**